go/server.go

```diff
@@ -261,29 +358,24 @@ func main() {
 
 	r.GET("/", func(c *gin.Context) {
 		c.HTML(http.StatusOK, "index.tmpl", gin.H{
-			"plaid_environment":        "sandbox", // Switch this environment
-			"plaid_public_key":         PLAID_PUBLIC_KEY,
-			"plaid_products":           PLAID_PRODUCTS,
-			"plaid_country_codes":      PLAID_COUNTRY_CODES,
-			"plaid_oauth_redirect_uri": PLAID_OAUTH_REDIRECT_URI,
-			"plaid_oauth_nonce":        PLAID_OAUTH_NONCE,
-			"item_id":                  itemID,
-			"access_token":             accessToken,
+			"item_id":        itemID,
+			"access_token":   accessToken,
+			"plaid_products": PLAID_PRODUCTS,
 		})
 	})
 
+	// For OAuth flows, the process looks as follows.
+	// 1. Create a link token with the redirectURI (as white listed at https://dashboard.plaid.com/team/api).
+	// 2. Once the flow succeeds, Plaid Link will redirect to redirectURI with
+	// additional parameters (as required by OAuth standards and Plaid)
```

Comment:
```suggestion
	// additional parameters (as required by OAuth standards and Plaid).
```
for consistency? : p